docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -24,10 +24,38 @@ Also requires <<privileges-list-indices,manage privileges>> on all indices that
 [[put-search-application-path-params]]
 ==== {api-path-parms-title}
 
+
 `create`::
 (Optional, Boolean) If `true`, this request cannot replace or update existing Search Applications.
 Defaults to `false`.
 
+`<body>`::
+(Optional, object)
+Contains parameters for a search application:
++
+.Properties of `<body>` objects
+[%collapsible%open]
+====
+`indices`::
+(Required, array of strings)
+The <<indices,indices>> associated with this search application. All indices need to exist in order to be added to a search application.
+
+`analytics_collection_name`::
+(Optional, string) The name of the behavioral analytics association being used with this search application.
+Presently, this is used for informational purposes only. For more information see: <<behavioral-analytics-apis
+
+`template`::
+(Optional, object)
+The <<search-template,search template>> associated with this search application. The search application's template is only stored and accessible through the search application.
+
+- This search template must be a Mustache template.
+- The template must contain a Mustache script and script source.
+- The template may be modified with subsequent <<put-search-application,put search application>> requests.
+- If no template is specified when creating a search application, or if a template is removed from a search application, we use the <<query-string-query-ex-request,query_string>> defined in the template examples as a default.
+- This template will be used by the <<search-application-search,search application search>> API to execute searches.
```

Comment:
I don't think search API is documented. Will that be done as part of https://github.com/elastic/enterprise-search-team/issues/3452, or should we create a separate issue?